tests/fixtures.py

```diff
@@ -20,6 +20,35 @@
 }
 """
 
+sample_toml = """
+ecs_name = "ardere-test"
```

Comment:
It's possible, though I was thinking the examples would be in the docs that will eventually be written.